dashboards-observability/public/components/visualizations/annotations/annotations.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+interface AnnotationProps {
+  showInputBox: boolean;
+  onTextChange: Function;
+  onAddAnnotation: Function;
+}
+
+export const Annotations: React.FC<AnnotationProps> = (props) => {
+  const handleAddAnnotation = () => {
+    props.onAddAnnotation();
+  };
+
+  const handleEditAnnotation = () => {};
+
+  const handleDeleteAnnotation = () => {};
+
+  return props.showInputBox ? (
+    <div style={{ display: 'flex', justifyContent: 'center' }}>
+      <input
+        type="text"
+        name="name"
+        placeholder="Add annotations"
+        onChange={(e) => props.onTextChange(e)}
+        style={{ padding: '5px' }}
+      />
+      <button
+        type="submit"
+        onClick={handleAddAnnotation}
+        style={{
```

Comment:
same as above and try to match button and input Ui to existing input and button.

Reply:
Done